### DISCUSSION TITLE
Help needed with default setup entities

### DISCUSSION THREAD

**Discussion by marekchowaniok** [sfstar/hass-victron#195] [Q&A]
Hi,  (sorry for basic question but I guess I need help)

this addon has provided to me lots of entities and I am not sure which one are the proper to use for "energy dashboard" . 
I went through all issues and searched for energy dashboard and try to see what other people used in their setup.  (I think  found one part of screenshot for electric grid)

I have put something together but frankly I am not sure. Can you share / help me which one to set ? ( JFYI - I got my solars up and running few hours ago and am excited but would need help to set up it correctly).  
Thank your for your help. 

<img width="1998" alt="image" src="https://github.com/sfstar/hass-victron/assets/404424/6921a389-1737-458d-a280-ae9a6e5236fb">

**Comment by skorunka** [comment]
@marekchowaniok Hello, any luck with the integration? I'm facing the same issue. Unfortunately, I did not find any explanation on what each sensor is for, so it is very hard for us beginners to do the HA Energy Dashboard integration. I was onyl able to configure the `Grid consumption` by creating this helper template (my setup has 3 phases/leads so I have to sum it up as HA expects just one value):

```
{% set l1 = states('sensor.victron_system_grid_l1') | float %}
{% set l2 = states('sensor.victron_system_grid_l2') | float %}
{% set l3 = states('sensor.victron_system_grid_l3') | float %}
{{ (l1 + l2 + l3 ) | float }}
```

![image](https://github.com/sfstar/hass-victron/assets/218913/888e0115-23e2-483a-8b3c-cda897a09589)

![image](https://github.com/sfstar/hass-victron/assets/218913/42c9cb29-d748-41b0-8b3d-40d1029e85a6)
